src/Service/MagicLinks/Models/SendMagicLinkRequest.cs

```diff
@@ -17,10 +18,10 @@ public class SendMagicLinkRequest
     [Required]
     public string UserId { get; init; }
 
-    public MagicLinkDTO ToDto() => new()
-    {
-        UserId = UserId,
-        EmailAddress = new MailAddress(EmailAddress),
-        LinkTemplate = UrlTemplate
-    };
+    /// <summary>
+    /// Number of seconds the magic link will be valid for.
+    /// </summary>
+    public int? TimeToLive { get; init; }
+
+    public MagicLinkTokenRequest ToDto() => new(UserId, new MailAddress(EmailAddress), UrlTemplate);
```

Comment:
maybe issue: You are not using the TimeToLive field.

Reply:
Fixed this.